package.json

```diff
@@ -5,51 +5,52 @@
   "scripts": {
     "start": "node lib/server.js",
     "stop": "pkill -SIGINT Hearth",
-    "dev:start": "NODE_ENV='development' npm run start",
+    "dev:start": "cross-env NODE_ENV='development' npm run start",
     "test": "npm run test:code && npm -s run test:style",
-    "test:code": "NODE_ENV='test' tap --cov --reporter=spec 'test/**/*.js'",
+    "test:code": "cross-env NODE_ENV='test' tap --cov 'test/**/*.js'",
```

Comment:
I'm not sure why the spec reporter needs to be removed? It produces a prettier output.

Reply:
Removing the spec reporter reduces the overhead required to print to the console. Since there are a lot of tests in Hearth, this improves time needed to run tests, which is helpful while developing. 

Also, I don't see much benefit having that output since it does not give us any further useful information which the error report would give us if any of the tests fail.

Reply:
Ok this reasoning seems fine.

Reply:
Ok, this sounds fine to me if there is a noted improvement in performance.